R/git.R

```diff
@@ -352,11 +341,14 @@ git_sitrep <- function(tool = c("git", "github"),
   if ("git" %in% tool && "user" %in% scope) {
     cli::cli_h3("Git global (user)")
     git_user_sitrep("user")
-    kv_line("Global (user-level) gitignore file", git_ignore_path("user"))
+    kv_line(
+      "Global (user-level) gitignore file",
+      I("{.path {git_ignore_path('user')}}")
```

Comment:
Occasionally the value of a kev-value line (which is what `kv_line()` handles) needs explicit inline styling and interpolation. Applying `AsIs` allows to me detect that. The default behaviour is to style the value as `{.val {value}}`.